NotebookAnalyzers/utils.py

```diff
@@ -27,7 +27,32 @@ def notebookToCode(filename):
     return py_code
 
 def functionsNumber(code):
-    """The function takes a python code sintax tree and returns the number of function definitions"""
+    """The function takes a python code and returns the number of function definitions"""
     tree = ast.parse(code)
     f_num=sum(isinstance(exp, ast.FunctionDef) for exp in tree.body)
-    return f_num
\ No newline at end of file
+    return f_num
+
+def importsCorrectPosition(code):
+    """The function takes a python code and returns True if there are no imports other the ones in the first cell of code, False otherwise"""
```

Comment:
The function takes a **Python code string** and returns True if there are no imports other **than those** in the first cell of code **and** False otherwise.